app/tests/hanging_protocols_tests/test_forms.py

```diff
@@ -294,3 +312,223 @@ def test_hanging_protocol_clientside():
+            ),
+        ),
+        (
+            1,
+            0,
+            1,
+            1,
+            (
+                "The following interfaces are used in your {}: test-ci-isolated-0, test-ci-image-0, and test-ci-undisplayable-0. "
+                'Example usage: {{"main": ["test-ci-isolated-0"], "secondary": ["test-ci-image-0"]}}. '
+                'Refer to the <a href="https://testserver/documentation/viewer-content/">documentation</a> for more information'
+            ),
+        ),
+        (
+            0,
+            1,
+            0,
+            1,
+            (
+                "The following interfaces are used in your {}: test-ci-overlay-0 and test-ci-undisplayable-0. "
+                "No interfaces of type (image, chart, pdf, mp4, thumbnail_jpg, thumbnail_png) are used. At least one interface of those types is needed to config the viewer. "
+                'Refer to the <a href="https://testserver/documentation/viewer-content/">documentation</a> for more information'
+            ),
+        ),
+        (
+            0,
+            1,
+            1,
+            1,
+            (
+                "The following interfaces are used in your {}: test-ci-isolated-0, test-ci-overlay-0, and test-ci-undisplayable-0. "
+                'Example usage: {{"main": ["test-ci-isolated-0"]}}. '
+                'Refer to the <a href="https://testserver/documentation/viewer-content/">documentation</a> for more information'
+            ),
+        ),
+        (
+            1,
+            1,
+            0,
+            1,
+            (
+                "The following interfaces are used in your {}: test-ci-image-0, test-ci-overlay-0, and test-ci-undisplayable-0. "
+                'Example usage: {{"main": ["test-ci-image-0", "test-ci-overlay-0"]}}. '
+                'Refer to the <a href="https://testserver/documentation/viewer-content/">documentation</a> for more information'
+            ),
+        ),
+        (
+            1,
+            1,
+            1,
+            1,
+            (
+                "The following interfaces are used in your {}: test-ci-isolated-0, test-ci-image-0, test-ci-overlay-0, and test-ci-undisplayable-0. "
+                'Example usage: {{"main": ["test-ci-isolated-0"], "secondary": ["test-ci-image-0", "test-ci-overlay-0"]}}. '
+                'Refer to the <a href="https://testserver/documentation/viewer-content/">documentation</a> for more information'
+            ),
+        ),
+    ),
+)
+@pytest.mark.parametrize(
+    "overlay_type",
+    (
+        InterfaceKindChoices.SEGMENTATION,
+        InterfaceKindChoices.STRING,
+    ),
+)
+@pytest.mark.parametrize(
+    "isolated_interface_type",
+    (
+        InterfaceKind.InterfaceKindChoices.CHART,
+        InterfaceKind.InterfaceKindChoices.PDF,
+    ),
+)
+@pytest.mark.django_db
+def test_forms_view_content_help_text(
```

Comment:
This is a comprehensive test due to the many cases to cover (images, overlays, isolated_interfaces, undisplayable_interfaces) but maybe it is better to split. 
To discuss

Reply:
Very complex indeed. You can also parametrize the object (Algorithm, Archive etc), which would reduce the actual assert part of the test to just a few lines. 

And you also want to have a test for an example that fails validation to make sure that your validation step is working.

Reply:
I parametrized the object but still there is a difference in the implementation between (Algorithm, Phase) and (Archive, ReaderStudy) so I had to use an `if` statement. It seemed a reasonable tradeoff compared to having two tests with lengthy case coverage.

Reply:
One more point, by trying to cover as much as possible of interface types, the test now unfold to 128 tests taking ~15 seconds to finish. We could for example do the tests with only one example of mandatory_isolated_interfaces and one of the overlays instead of currently testing with two of each. That will bring the number of tests down to 32 tests taking ~5 sec.

Reply:
I think here the only thing you want to test is that the helptext is added to the field. The actualy content (i.e. the example generation) is tested in the next test, right? So then you can just test the case where there are no interfaces, or none of the necessary kind and one case where the view content is generated. Or did I misunderstand?

Reply:
That is correct, the next test does the example check. For this one, we need to test the following cases:
- no interfaces at all
- no interfaces of type Image or isolated, but others like overlays or undisplayable (so we cover the case where we have both messages "you have the following interfaces .." and "No interfaces of type".
- one case where the view content is generated

Done